test/inductor/test_fp8.py

```diff
@@ -93,6 +114,36 @@ def fp8_cast(x, dtype):
             y = compiled_fp8_cast(x, torch.float8_e4m3fn)
 
 
+    @unittest.skipIf(TEST_WITH_ROCM, "FP8 is not supported on ROCM")
+    @unittest.skipIf(not SM90OrLater, "FP8 is only supported on H100+")
+    @parametrize("float8_dtype", (torch.float8_e4m3fn, torch.float8_e5m2))
+    def test_layernorm_fp8_quant(self, float8_dtype: torch.dtype):
+        hidden_size = 4096
+        sequence_length = 2048
+        batch_size = 4
+
+        def ln_fp8(x: Tensor, scale: float, amax_buffer: Tensor):
+            x = torch.nn.functional.layer_norm(x, [hidden_size], weight=None, bias=None, eps=1e-05)
+            amax_buffer.fill_(torch.max(torch.abs(x)))
```

Comment:
if it helps, this function can be moved to after the pointwise stuff in user code.  The way we originally wrote this code isn't super intuitive

Reply:
I feel it doesn't matter. This requires a reduction kernel anyways.